packages/shared/src/types/abTests/epic.ts

```diff
@@ -153,40 +145,20 @@ export interface AmountsTest {
 
 export type AmountsTests = AmountsTest[];
 
-export interface EpicTest extends Test<EpicVariant> {
-    name: string;
-    status: TestStatus;
-    locations: CountryGroupId[];
-    tagIds: string[];
-    sections: string[]; // section IDs
-    excludedTagIds: string[];
-    excludedSections: string[];
-    alwaysAsk: boolean;
-    maxViews?: MaxViews;
-    userCohort: UserCohort;
-    hasCountryName: boolean;
-    variants: EpicVariant[];
-    highPriority: boolean;
-    useLocalViewLog: boolean;
-    articlesViewedSettings?: ArticlesViewedSettings;
-    hasArticleCountInCopy: boolean; // added by the server - we do not want to determine this for each client request
-
-    audience?: number;
-    audienceOffset?: number;
-
-    // These are specific to hardcoded tests
-    expiry?: string;
-    campaignId?: string;
-
-    controlProportionSettings?: ControlProportionSettings;
+// for validation from DynamoDB
+export type EpicTest = z.infer<typeof EpicTestSchema>;
 
-    // added by the server
+// with additional properties determined by the server
+export interface EpicTestProcessed extends EpicTest, Test<EpicVariant> {
+    hasArticleCountInCopy: boolean;
     isSuperMode?: boolean;
+    variants: EpicVariant[];
```

Comment:
why is it necessary to define the variants field here?

Reply:
I added that to satisfy Typescript - there are fields in EpicVariant that aren't defined in variantSchema
error: "Named property variants of types: _ and `Test<EpicVariant>` are not identical"